docs/en/observability/observability-introduction.asciidoc

```diff
@@ -1,57 +1,30 @@
 [[observability-introduction]]
 [role="xpack"]
-== What is Elastic Observability? 
-
-coming[a future update]
-//TODO: Add content
-
-
-[[observability-get-started]]
-== Get started with the {stack}
-
-coming[a future update]
-
-To get started...
-
-=== Hosted Elasticsearch Service
-
-coming[a future update]
-
-=== Install the {stack} yourself
-
-coming[a future update]
-
-=== Run the {stack} on Docker
-
-coming[a future update]
-
-=== Run the {stack} on Kubernetes
-
-coming[a future update]
-
-[float]
-==== Step 1: Install {es}
-
-[float]
-==== Step 2: Install {kib}
-
-[[observability-add-data]]
-== Add observability data 
-
-coming[a future update]
-
-=== Instrument applications
-
-coming[a future update]
-
-=== Ingest logs
-
-coming[a future update]
-
-=== Ingest metrics
-
-coming[a future update]
-
-=== Ingest uptime data
-
-coming[a future update]
+= What is Elastic Observability? 
+
+Observability provides you with granular insights and context into the behaviour
+of applications running in your environments. At Elastic, we view observability as an
+attribute of any system that you build and want to monitor. Being able to detect
+and fix root cause events quickly within an observable system is what we consider
+a minimum requirement for any analyst.
+
+Do you face the difficulty of ingesting metrics into one system and logs into 
+another system? Are you also using a separate APM tool? Are you finding that manual
+correlation of these various data sources is time consuming and maintenance is a 
+significant operational cost?
+
+With https://www.elastic.co/observability[Elastic Observability], we provide you with a 
+single stack to unify your logs, system metrics, uptime data, and application traces. 
+Each one of your ingested data
+sources creates an index in https://www.elastic.co/elasticsearch[{es}], with no restrictions on:
+
+- aggregations that you run across all of your data
+- how data is displayed in https://www.elastic.co/kibana[{kib}]
+- how alerting and machine learning applies to each data source
+
+This unified approach provides a comprehensive overview and proactive management of all your data.
+You can search, monitor, and apply analytics in real time to events happening across all of
+your environments.
+
+[role="screenshot"]
+image::images/observability.png[Elastic Observability]
```

Comment:
This image is OK, but it'd be great to get something a bit more custom for here

Reply:
Let me see what I can come up with.